### DISCUSSION TITLE
Tastaturgenveje virker ikke

### DISCUSSION THREAD

**Discussion by obeejohnkanobi** [Eduap-com/WordMat#64] [Support] [answered]
Hej

Efter den nye opdatering virker alt m, alt b, alt L, ikke, både til windows word og word på mac.

**Comment by mathiasfugmann** [comment, marked as answer]
Har du prøvet at genstarte WordMat ved at gå til WordMats indstillinger under 'Avanceret'? Det har virket for mig selv og dem, jeg har hjulpet.

**Comment by obeejohnkanobi** [comment]
Den kommentar er der rigtigt mange i min klasse der vil blive rigtigt glade for :) 

Tak for hjælpen. Det hjalp.

**Comment by NikolajHenriksen** [comment]
Den her tråd er godt nok gammel, men jeg oplever netop problemet ovenfor. Det løses dog ikke ved at "Genstart WordMat" eller ved "Reparer genvejstaster".
Andre der har stadig oplever problemet?
